convex_hull.cpp

```diff
@@ -178,13 +181,13 @@ int main(int argc, char* argv[]) {
 
 	convex_hull<double> list;
 
-	long long int cases;
-	if(scanf("%lld",&cases) != 1)
+	long long int no_of_points;
+	if(scanf("%lld",&no_of_points) != 1)
 	{
 	  	fprintf(stderr, "Error, please enter a valid Integer.\n");
```

Comment:
Correct this message to something like `File is malformed. It should have two space separated real numbers per line`.